routes/v1/reports.js

```diff
@@ -0,0 +1,41 @@
+import express from 'express';
+import Middleware from '../../middlewares';
+import Report from '../../controllers/reports';
+import Validation from '../../validators/articles';
+
+
+const router = express.Router();
+
+router.post(
+  '/',
+  Middleware.authenticate,
+  Middleware.isblackListedToken,
+  Validation.reportArticle,
+  Report.reportArticle
+);
+
+router.get(
+  '/articles',
+  Middleware.authenticate,
+  Middleware.isblackListedToken,
+  Middleware.isAdmin,
+  Report.getReportedArticles
+);
+
+router.get(
+  '/:id/article',
```

Comment:
Good job on this PR. Following the convention from the other routes, it is better you use article slug in your params rather than the article id.

Reply:
thank you